src/theme/components/multiSelectItemsCountButton.ts

```diff
@@ -0,0 +1,39 @@
+const MultiSelectItemsCountButton = {
+  baseStyle: ({ isOpen = false }) => ({
+    alignItems: "center",
+    backgroundColor: "ui.gray.light-cool",
+    border: "1px",
+    borderRadius: "20px",
+    borderColor: isOpen ? "ui.border.hover" : "ui.border.default",
+    display: "flex",
+    flexShrink: 0,
+    fontSize: "desktop.caption",
+    justifyContent: "flex-end",
+    left: "15px",
+    marginRight: "xs",
+    position: "absolute",
+    top: { base: "12px", md: "10px" },
+    width: "46px",
+    zIndex: 10000,
```

Comment:
@EdwinGuzman I am not happy with this solution, but this is the only thing I could come up with to solve the layering issue.  Focus makes the main button jump to `9999`, so I simply set the "items count" button to `10000`.  Can you think of other possible solutions?

Reply:
Oh, are we putting the 9999 value on focus? Let's leave this as 1000, it's not a problem but we should track all the z-index rules we have.

Reply:
Yes, `zIndex: "9999"` is part of the default focus styles.